e2e/test/scenarios/models/reproductions/19776-data-picker-not-displayed-after-archiving-model.cy.spec.js

```diff
@@ -1,8 +1,9 @@
-import { restore, popover } from "e2e/support/helpers";
+import { restore, popover, describeOSS } from "e2e/support/helpers";
 
 const modelName = "Orders Model";
 
-describe("issue 19776", () => {
+// this is only testable in OSS because EE always has models from auditv2
+describeOSS("issue 19776", { tags: "@OSS" }, () => {
```

Comment:
@nemanjaglumac , a couple questions

- is `describeOSS` deprecated? if so, we should just remove it, right?
- does adding the `@OSS` tag here to make sure it only runs on a version of metabase without audit models make sense to you? Anything I'm missing here?

Reply:
- `describeOSS` will skip tests if Metabase version is not OSS.
- Tag `@OSS` doesn't guarantee anything unless you remember to use it in a Cypress run command.

Does this make sense?
And does it answer your question?